{
    "type": "bundle",
    "id": "bundle--a6fc319b-7c97-4c9b-94d4-94f72593d177",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--2bee6b04-4b71-4a00-b8fa-fbaca1d3d30b",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-03T00:39:49.614242Z",
            "modified": "2025-04-03T00:39:49.614242Z",
            "name": "CVE-2025-2998",
            "description": "A vulnerability was found in PyTorch 2.6.0. It has been declared as critical. Affected by this vulnerability is the function torch.nn.utils.rnn.pad_packed_sequence. The manipulation leads to memory corruption. Local access is required to approach this attack. The exploit has been disclosed to the public and may be used.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-2998"
                }
            ]
        }
    ]
}